move-forks-to-organization.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+function printError() {
+	echo "$@"
+	exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+	name="$(basename "$0")"
+	printError "Usage: $name <user> <newowner>"
+fi
+
+user=$1
+newowner=$2
+
+which jq >/dev/null || printError "please install jq"
+which gh >/dev/null || printError "please install GitHub CLI"
```

Comment:
On some systems/shells, the `which` command will return 0 exit code regardless of the existance of the command.
Prefer using the `command` builtin instead..

```bash
command -v <CMD> >/dev/null || printError "..."
```

Reference: https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script

Reply:
Yez, indeed I codes quickly.

I'll iterate,that's why I opened a PR

Reply:
I will merge now, but I will address your feedback with:

- https://github.com/ccoVeille/github-forks-sanitizer/issues/4